galicaster/utils/ical.py

```diff
@@ -85,17 +116,34 @@ def create_mp(repo, event):
     mp.setTitle(event['SUMMARY'])
     mp.setDate(event['DTSTART'].dt.replace(tzinfo=None))
 
+    catalog= mp.getCatalogs("dublincore/series")
+    if catalog:
+        mp.remove(catalog[0])
+
+    if 'identifier' in mp.metadata_series:
+        mp.metadata_series["identifier"] = None
+        mp.metadata_series['title'] = None
+    if "isPartOf" in mp.metadata_episode:
+        del mp.metadata_episode['isPartOf']
+    if 'creator' in mp.metadata_episode:
+        mp.metadata_episode['creator'] = None
+
+
+
     # ca_properties_name = 'org.opencastproject.capture.agent.properties'
     for attach_enc in event['ATTACH']:
         attach =  base64.b64decode(attach_enc)
         if attach_enc.params['X-APPLE-FILENAME'] == 'episode.xml':
+            if not rewrite1:
```

Comment:
This means that when calling `create_mp(repo, event)` as before, the `rewrite` value will be equal to `None`, since `rewrite1` is `None` by default, even if this:
```
if repo.has_key(event['UID']):
```
is true (see [ical.py line 103](https://github.com/teltek/Galicaster/blob/2b2be5529b00b131b704138f92ef94be671937bd/galicaster/utils/ical.py#L103))

If this is intentional, there should be a more clear way implement it.